src/AnalyzerPowerPack/Core/Usage/SerializationRulesDiagnosticAnalyzer.cs

```diff
@@ -13,59 +13,84 @@ public sealed class SerializationRulesDiagnosticAnalyzer : DiagnosticAnalyzer
+
+        private static readonly LocalizableString s_localizableTitleCA2235 =
+            new LocalizableResourceString(nameof(AnalyzerPowerPackRulesResources.MarkAllNonSerializableFields),
+                AnalyzerPowerPackRulesResources.ResourceManager, typeof (AnalyzerPowerPackRulesResources));
+
+        private static readonly LocalizableString s_localizableMessageCA2235 =
+            new LocalizableResourceString(nameof(AnalyzerPowerPackRulesResources.FieldIsOfNonSerializableType),
+                AnalyzerPowerPackRulesResources.ResourceManager, typeof (AnalyzerPowerPackRulesResources));
+
+        private static readonly LocalizableString s_localizableDescriptionCA2235 =
+            new LocalizableResourceString(
+                nameof(AnalyzerPowerPackRulesResources.MarkAllNonSerializableFieldsDescription),
+                AnalyzerPowerPackRulesResources.ResourceManager, typeof (AnalyzerPowerPackRulesResources));
 
         internal static DiagnosticDescriptor RuleCA2235 = new DiagnosticDescriptor(RuleCA2235Id,
-                                                                         s_localizableTitleCA2235,
-                                                                         s_localizableMessageCA2235,
-                                                                         AnalyzerPowerPackDiagnosticCategory.Usage,
-                                                                         DiagnosticSeverity.Warning,
-                                                                         isEnabledByDefault: true,
-                                                                         description: s_localizableDescriptionCA2235,
-                                                                         helpLinkUri: "http://msdn.microsoft.com/library/ms182349.aspx",
-                                                                         customTags: DiagnosticCustomTags.Microsoft);
+            s_localizableTitleCA2235,
+            s_localizableMessageCA2235,
+            AnalyzerPowerPackDiagnosticCategory.Usage,
+            DiagnosticSeverity.Warning,
+            isEnabledByDefault: true,
+            description: s_localizableDescriptionCA2235,
+            helpLinkUri: "http://msdn.microsoft.com/library/ms182349.aspx",
+            customTags: DiagnosticCustomTags.Microsoft);
 
-        private static readonly ImmutableArray<DiagnosticDescriptor> s_supportedDiagnostics = ImmutableArray.Create(RuleCA2229, RuleCA2235, RuleCA2237);
+        private static readonly ImmutableArray<DiagnosticDescriptor> s_supportedDiagnostics =
+            ImmutableArray.Create(RuleCA2229, RuleCA2235, RuleCA2237);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
```

Comment:
Instead use expression bodied member: => s_supportedDiagnostics and that can be inlined as well and you can get rid of the field